### DISCUSSION TITLE
Precision Farming

### DISCUSSION THREAD

**Discussion by MaddyPhi** [Royal-Modding/FS19_CallContractors#11] [General]
Hi, I am enjoying the mod but I am having an issue with it using precision farming. When I request a liming contractor, the liming is performed as evidenced by the white field but the ph levels are unchanged. Is it possible to make this work with precision farming or is that something that is not possible?

**Comment by TyKonKet** [comment]
At the moment we have not been able to make it work, but we are still trying to improve it.
